pkg/db/postgres/run/run.go

```diff
@@ -560,14 +563,18 @@ func (m *runPG) find(
 		)
 		select "run_id"
 		from "assign_and_data"
+		where 
+			($9::timestamp with time zone is null or "updated_at" >= $9::timestamp with time zone)
+			and ($10::timestamp with time zone is null or "updated_at" <= $10::timestamp with time zone)
```

Comment:
Isn't it `<` ?

```suggestion
			and ($10::timestamp with time zone is null or "updated_at" < $10::timestamp with time zone)
```

I'm uncertine about this. Users should be able to query without overlapping by

```
knit run find --duration 24h --since 2024-01-01
knit run find --duration 24h --since 2024-01-02
```

Is this SQL doing that?